waiter/src/waiter/token.clj

```diff
@@ -63,23 +65,29 @@
           (when owner
             (let [owner-key (ensure-owner-key kv-store owner->owner-key owner)]
               (update-kv kv-store owner-key (fn [v] (conj (or v #{}) token)))))
-          (log/info "Stored service description template for" token)))))
+          (log/info "stored service description template for" token)))))
 
   (defn delete-service-description-for-token
     "Delete a token from the KV"
-    [synchronize-fn kv-store token owner]
+    [clock synchronize-fn kv-store token owner & {:keys [excise] :or {excise false}}]
     (synchronize-fn
       token-lock
       (fn inner-delete-service-description-for-token []
-        (log/info "Deleting service description for token:" token)
-        (kv/delete kv-store token)
+        (log/info "deleting service description for token:" token "excise mode:" excise)
```

Comment:
changed log to read `attempting to delete service...`.

The kv implementation does not currently notify whether a delete was a no-op.